### ISSUE TITLE
Script for plotting learning curves

### ISSUE THREAD

**Issue by libornovax** [libornovax/master_thesis_code#14]
Read the output.txt, which I create during each training and extract from it the values of the loss function for training and validation set. Then plot them in a graph and save as PDF.
